client/style.sass

```diff
@@ -309,13 +307,17 @@ p.eventInfo
 
 #createEventZipcode  
   width: 100%
-  margin-bottom: 10pxx
+  margin-bottom: 10px
 
 #createEventDescription
   width: 100%
   margin-bottom: 10px
   height: 150px
 
+#Username
```

Comment:
I don't think this actually does anything. I think you did the styling in the...